draft-ietf-privacypass-architecture.md

```diff
@@ -198,30 +198,40 @@ The end-to-end flow for Privacy Pass involves three different types of
 contexts:
 
 - Redemption context: The interactions and set of information shared
-between the Client and Origin.
+between the Client and Origin. This context includes all information
+associated with redemption, such as the timestamp of the event, Client
+visible information (including the IP address), and the Origin name.
 - Issuance context: The interactions and set of information shared
-between the Client, Attester, and Issuer.
+between the Client, Attester, and Issuer. This context includes all
+information associated with issuance, such as the timestamp of the event,
+any Client visibile information (including the IP address), and the
+Origin name (if revealed during issuance).
 - Attestation context: The interactions and set of information shared between
 the Client and Attester only, for the purposes of attesting the vailidity of
-the Client.
+the Client. This context includes all information associated with attestation,
+such as the timestamp of the event and any Client visibile information,
+including the IP address or other information specific to the type of
```

Comment:
the wording here seems a little odd, maybe implying IP address and other information is necessarily visible?

Reply:
Rephrased slightly to focus just on attestation information.